app/sidekiq/send_notif_slack_when_add_value_job.rb

```diff
@@ -0,0 +1,40 @@
+class SendNotifSlackWhenAddValueJob
+  include Sidekiq::Job
+  sidekiq_options retry: false
+
+  def perform project_id, month, year
+    client = Slack::Web::Client.new
+
+    project = Project.find_by id: project_id
+    sum_man_month = project.project_features.filter_month(month)
+                           .filter_year(year).sum(:man_month)
+
+    project_slacks = project.project_slack_settings
+    project_slacks.each do |project_slack|
+      next unless project_slack.send_value?
+
+      mention = convert_to_slack_mention project_slack.slack_mention
+
+      client.chat_postMessage(
+        channel: project_slack.slack_room,
+        text: message(project.name, mention,
+                      month, year,
+                      sum_man_month),
+        as_user: true
+      )
+    end
+  end
+
+  def message project_name, mention, month, year, sum_man_month
+    I18n.locale = :vi
+    "#{mention}\n
+     #{I18n.t('add_value.message.project_name')}: #{project_name}\n
```

Comment:
```suggestion
     #{I18n.t('add_value.message.project_name', locale: :vi)}: #{project_name}\n
```
e làm ntn nhé. chứ dòng trên là sửa i18n của hệ thống luôn r